EIPS/eip-1901.md

```diff
@@ -0,0 +1,66 @@
+# Add OpenRPC Service Discovery To JSON-RPC Services
+
+    EIP: ??
+    Title: Add OpenRPC Service Discovery To JSON-RPC Services
+    Author: Shane Jonas, Zachary Belford
```

Comment:
Should include email or a github username to at least one of the authors to support automerging in draft stage.